android/quest/src/test/java/org/smartregister/fhircore/quest/QuestApplicationTest.kt

```diff
@@ -39,6 +44,20 @@ class QuestApplicationTest : RobolectricTest() {
     application.workerFactory = mockk()
   }
 
+  @Test
+  fun testSentryMonitoringWhenDsnNotBlank() {
+    val sentryDsn = "debb3087-167a-47ff-b6d4-737be3965a4c"
+    val spyApp = spyk(application)
+    spyApp.initSentryMonitoring(dsn = sentryDsn)
+  }
+
+  @Test
+  fun testSkipSentryMonitoringWhenDsnBlank() {
+    val sentryDsn = ""
+    val spyApp = spyk(application)
+    spyApp.initSentryMonitoring(dsn = sentryDsn)
+  }
```

Comment:
My changes cause these tests to fail for now. If we want to mock, I think we should mock inside SentryAndroid.init, or (not great) we could have that be a default and pass in a mockk for it, but I'd worry that approach would hide problems